tests/interface/TWBase32Tests.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "TWTestUtilities.h"
+
+#include "Data.h"
+#include <TrustWalletCore/TWBase32.h>
+
+#include <gtest/gtest.h>
+
+TEST(TWBase32, InvalidDecode) {
+    const auto encodedInput = STRING("JBSWY3DPK5XXE3DE=======");
+    auto result = WRAPD(TWBase32Decode(encodedInput.get()));
+    ASSERT_EQ(result, nullptr);
```

Comment:
consider adding some blank line before `ASSERT_EQ` easier to read:
// assign statement
// assign statement

// assert / eq

// another assign / test section
```suggestion`

    ASSERT_EQ(result, nullptr);
```